exporter/otlp-metrics/lib/opentelemetry/exporter/otlp/metrics/metrics_exporter.rb

```diff
@@ -214,11 +214,23 @@ def encode(metrics_data)
           end
 
           # metrics_pb has following type of data: :gauge, :sum, :histogram, :exponential_histogram, :summary
-          # current metric sdk only implements instrument: :counter -> :sum, :histogram -> :histogram
+          # current metric sdk only implements instrument: :counter -> :sum, :histogram -> :histogram, :gauge -> :gauge
           #
           # metrics [MetricData]
           def as_otlp_metrics(metrics)
             case metrics.instrument_kind
+            when :gauge
```

Comment:
Is that possible to combine `:observable_gauge` and `:gauge` something like `when :observable_gauge, :gauge` since they are using same `Proto::Metrics::V1::Gauge`.